BSDmakefile

```diff
@@ -0,0 +1,11 @@
+# pmake might add -J (private)
+FLAGS=${.MAKEFLAGS:C/\-J ([0-9]+,?)+//W}
+
+all: .DEFAULT
+.DEFAULT:
+	@which gmake > /dev/null 2>&1 ||\
+		(echo "GMake is required for node.js to build.\
+			Install and try again" && exit 1)
+	@gmake ${.FLAGS} ${.TARGETS}
+
+.PHONY: test
```

Comment:
I'm not sure we need this and prefer to avoid maintaining the BSDMakefile (even if it's just a redirect).